src/muiThemeable.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
I'd say wait a bit since IDEs get smart enough. the jsx syntax was giving me a hard time over this!